bundle.Dockerfile

```diff
@@ -6,9 +6,9 @@ LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=service-telemetry-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=unstable
 LABEL operators.operatorframework.io.bundle.channel.default.v1=unstable
-LABEL operators.operatorframework.io.metrics.mediatype.v1=metrics+v1
 LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v0.19.4
 LABEL operators.operatorframework.io.metrics.project_layout=ansible
+LABEL operators.operatorframework.io.metrics.mediatype.v1=metrics+v1
```

Comment:
any reason to move away from alphabetical ordering?

Reply:
The `operator-sdk generate bundle` command manages that file and seems to move things around periodically. I mostly just go with the flow because it'll move it somewhere else next time I run the command :)